js/apps/account-ui/public/content.json

```diff
@@ -0,0 +1,26 @@
+[
+  { "label": "personalInfo", "path": "/" },
+  {
+    "label": "accountSecurity",
+    "children": [
+      { "label": "signingIn", "path": "account-security/signing-in" },
+      { "label": "deviceActivity", "path": "account-security/device-activity" },
+      {
+        "label": "linkedAccounts",
+        "path": "account-security/linked-accounts",
+        "isHidden": "isLinkedAccountsEnabled"
```

Comment:
`isHidden` should be renamed to `isVisible` so that the reader of content.json understands it will be visible when `isLinkedAccountEnabled` is true and hidden when it is false.